master/buildbot/reporters/bitbucket.py

```diff
@@ -124,14 +124,18 @@ def get_owner_and_repo(repourl):
         Takes a git repository URL from Bitbucket and tries to determine the owner and repository
         name
         :param repourl: Bitbucket git repo in the form of
-                    git@bitbucket.com:OWNER/REPONAME.git
-                    https://bitbucket.com/OWNER/REPONAME.git
-                    ssh://git@bitbucket.com/OWNER/REPONAME.git
+                    git@bitbucket.org:OWNER/REPONAME.git
+                    https://bitbucket.org/OWNER/REPONAME.git
+                    ssh://git@bitbucket.org/OWNER/REPONAME.git
+                    https://api.bitbucket.org/2.0/repositories/OWNER/REPONAME
         :return: owner, repo: The owner of the repository and the repository name
         """
         parsed = urlparse(repourl)
 
-        if parsed.scheme:
+        base_parsed = urlparse(_BASE_URL)
```

Comment:
I think we should use self.base_url here.

Reply:
The method was static. I had to do some more changes. I'm not sure about the test. I hope I did it right.